### ISSUE TITLE
Feature image block with srcset

### ISSUE THREAD

**Issue by vyskoczilova** [soflyy/oxygen-bugs-and-features#1204]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*
When inserting a featured image as dynamic data, there is no srcset attribute etc, which is kind of shame.

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*

Every single post with image.

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*

Native WP

**Comment by yankiara** [comment]
+1000, MAJOR ISSUE HERE.

**Comment by githrun** [comment]
Yes this would be a feature we really need to fix speed issues on mobile.  For all images in oxygen to use SRCSET by default, including all background images.

**Comment by LauGau** [comment]
So yeah, I just saw that srcset doesn't work on dynamic data (like thumbnails)...
It's really a shame for my favorite anti-bloat builder. 😢

**Comment by thomasjulienalain** [comment]
+1

**Comment by wpsumo** [comment]
+1

**Comment by kevingeary** [comment]
+1

**Comment by adrien-robert** [comment]
+1

**Comment by yankiara** [comment]
Quick fix... use a code block with:

```
<?php
	the_post_thumbnail( 'your-max-size', [ 'class' => 'your classes' ] );
?>

```
A shame that dev team has not fixed that after one year. But we are used to it now :-(

**Comment by heavestudio** [comment]
> Quick fix... use a code block with:
> 
> ```
> <?php
> 	the_post_thumbnail( 'your-max-size', [ 'class' => 'your classes' ] );
> ?>
> ```
> 
> A shame that dev team has not fixed that after one year. But we are used to it now :-(

Did that but it seems that only one size is loaded.

**Comment by derSebastian** [comment]
Still an issue in v4.0.2 RC 1.

Please put this closer to the top of the list of bug fixes for 4.0.3?

**Comment by JakubJeziorski** [comment]
Can we have this fixed, please?
